src/utils/update/mapper.utils.ts

```diff
@@ -68,6 +68,7 @@ export const mapAddress = (address: ServiceAddress | undefined): Address => {
 type BOMOAddressMapTypes = {
   (address: PSCAddress | undefined): Address;
   (address: OfficerAddress | undefined): Address;
+  (address: PrivateADDRESS | undefined): PrivateADDRESS;
```

Comment:
I'd prefer the SDK to have PrivaetAddress so this could still change

Reply:
oh, you mean without caps for Address, got it